tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -32,9 +33,13 @@ export const createUser = async ({ user, admin }: { user: User; admin: User }):
 
   checkResponseStatus(response, 'Failed while creating user')
 
+  const userCopy = { ...user }
+
   const responseData = await response.json()
-  user.uuid = responseData.id
+  userCopy.uuid = responseData.id
 
+  const usersEnvironment = new UsersEnvironment()
+  usersEnvironment.storeUser({ key: userCopy.id, user: userCopy })
```

Comment:
Could this be simplified to:
 usersEnvironment.storeUser({ key: responseData.id, user: {...user} })